src/components/input-field/input-field.tsx

```diff
@@ -371,6 +371,8 @@ export class InputField {
             <input
                 {...properties}
                 type={type}
+                readonly={this.readonly}
+                disabled={this.disabled}
```

Comment:
I assume you've added these to avoid setting `disabled` when `readonly` is set, but setting `disabled` along with `readonly` is intentional, so this addition should be removed.

Reply:
Why? 🤔  I thought we added support for both

Reply:
We do have support for both. The behaviour is different if you set `disabled` or `readonly`, but if you set both, it's the same as when you only set `readonly`. The native elements have no concept of "readonly", only "disabled". So our "readonly" is basically a specific styling of a disabled field, used to indicate to the user that "the reason you can't edit this field is of a more permanent nature than if it was just disabled".

Reply:
I thought we started to add support for that so I thought it was a good thing to set them

Reply:
`limel-input-field` already has support for both 🙂

As far as the consumer is aware, disabled and readonly are different things. But _internally_, our `readonly` state sets the `disabled` state on the native element used. That makes sense, since we don't want readonly inputs to be enabled.